eox_tagging/test/test_viewset.py

```diff
@@ -68,13 +68,50 @@ def test_create_tag(self):
             "tag_value": "example_tag_value",
             "target_type": "user",
             "target_id": "user_test",
-            "owner_type": "user",  # default is site
+            "owner_type": "user",
             "access": "PRIVATE",
-            "expiration_date": "2020-12-04 10:20:30"
+            "expiration_date": "2020-12-04 10:20:30",
         }
+
         response = self.client.post("/api/v1/tags/", data, format='json')
+
         self.assertEqual(response.status_code, 201)
 
+    def test_create_tag_with_iso_datetime_format(self):
+        """"Used to test creating a tag using ISO format in datetime fields."""
+        data = {
+            "tag_type": "example_tag_1",
+            "tag_value": "example_tag_value",
+            "target_type": "user",
+            "target_id": "user_test",
+            "owner_type": "user",
+            "access": "PRIVATE",
+            "expiration_date": "2020-12-04T10:20:30.15785",
+        }
+
+        response = self.client.post("/api/v1/tags/", data, format='json')
```

Comment:
reverse

Reply:
I fixed this in #16, should I rebase this PR with that one?

Reply:
that pr was already approved, please squash, merge and rebase this one

Reply:
Right. Thank you :)